execution_engine2.spec

```diff
@@ -314,29 +314,37 @@
 
         /*
             job_states - states of jobs
+
+            could be mapping<job_id, JobState> or list<JobState>
         */
         typedef structure {
-            mapping<job_id, JobState> job_states;
+            UnspecifiedObject job_states;
```

Comment:
Shouldn't list be a  list<JobStates> ?

Reply:
Oh I see.. It can return both...

Reply:
But then this breaks what @briehl said in https://github.com/kbase/execution_engine2/issues/49

Reply:
yea. I updated it as `list` since it's the default return type now.